shared_model/packages/javascript/README.md

```diff
@@ -0,0 +1,68 @@
+# iroha-js
+
+Official Iroha JavaScript Library. https://github.com/hyperledger/iroha
+
+## Usage
+
+You can use regular Node.js style to import **iroha-lib** package and related protobufs:
+
+```javascript
+const iroha = require('iroha-lib')
+
+const blockTransaction = require('iroha-lib/pb/block_pb.js').Transaction
+const endpointGrpc = require('iroha-lib/pb/endpoint_grpc_pb.js')
+
+...
+
+```
+
+Watch usage in *example* folder.
+
+## Build
+
+You need this section if you want to build **iroha-lib** manually for publishing or if your architecture/OS not supported yet.
+
+### Prerequisities
+
+In order to build NPM package by `node-gyp` on your machine you need some global installed dependencies: 
+
+1. CMake (>=3.7)
+
+2. Protobuf (>3.5)
```

Comment:
AFAIK we use **Protobuf** v3.5.0 and above because I got some errors with incompatible syntax when tried to generate *shema* interfaces by the old versions.

Reply:
Actually we use 3.4 in findprotobuf.cmake so that's quite strange. This error could appear if you mix protobuf versions.